pkg/minikube/cluster/cluster_test.go

```diff
@@ -417,3 +418,30 @@ func TestCreateSSHShell(t *testing.T) {
 		t.Fatalf("Expected ssh session to be run")
 	}
 }
+
+type MockServiceGetter struct{}
```

Comment:
One improvement here might be to make this:

type MockServiceGetter struct {
  map[string] api.Service
}

then make a constructor:

func NewMockServiceGetter() MockServiceGetter

that initializes the map and returns an instance. Then your test can create a new instance, and add the service into the map.

Then the Get function could just lookup the service from the map and return it if it exists.

That way we could use this in other places too, and you could add a second test case that tests when the service doesn't exist yet.